docs/20_samples/10_sample-edition/images/README.md

```diff
@@ -0,0 +1,11 @@
+# Images
+
+In this view, you can visualize images linked to your sample. You can either select an image proposed in the `Images` tab, or drag and drop your own image. The image formats tiff, png, gif and jpeg are supported. 
+
+All the images can be downloaded using the ![download](download.png) icon. 
+
+:::info 
+
+The definition of original image is conserved in the view.
```

Comment:
could you clarify this? It is unclear if you mean resolution, metadata or something else